openpype/hosts/nuke/api/lib.py

```diff
@@ -896,7 +896,9 @@ def get_view_process_node():
     ipn_orig = None
     for v in nuke.allNodes(filter="Viewer"):
         ipn = v['input_process_node'].getValue()
-        if "VIEWER_INPUT" not in ipn:
+        if ipn:
+            if "VIEWER_INPUT" in ipn:
+                return
```

Comment:
Note: I know nothing of the nuke codebase.

This seems odd. Why would it iterate over all of them but return early as soon as it finds on viewer with an input process node with "VIEWER_INPUT" in it. Are you sure you didn't mean `continue` instead of `return`? Or maybe `return ipn`?

Additionally, this function really needs a docstring to explain what it does.

Reply:
huh.. Well, that answer me the question `Is it good to code paralelly with client's weekly meeting?`
Fixed now.. Thanks!